### DISCUSSION TITLE
How to output relative paths?

### DISCUSSION THREAD

**Discussion by tkossak** [sharkdp/fd#1321] [Q&A] [answered]
Why this command doesn't output relative paths (without `/tmp` prefix)?
`fd -g '*' /tmp --base-directory /tmp --relative-path`

**Comment by tavianator** [comment, marked as answer]
You can remove the first `/tmp` to get your desired output:

```console
fd -g '*' --base-directory /tmp
```

`--relative-path` is the default so you don't need to specify it explicitly.

**Comment by tkossak** [comment]
Thank you very much!
Btw. do you know if fd has any option to remove trailing slashes on folder results? So far I was just piping the results to `... | sed 's/\/$//'` but maybe fd can already do it...

**Reply by tavianator** [reply]
There's no option for it, sed is probably your best bet.  You can write `sed 's|/$||'` so you don't have to escape the slash if you want.
